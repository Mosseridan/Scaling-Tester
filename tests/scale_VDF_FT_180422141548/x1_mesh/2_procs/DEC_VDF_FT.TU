Statistiques de post resolution

Temps total                       0.246051

