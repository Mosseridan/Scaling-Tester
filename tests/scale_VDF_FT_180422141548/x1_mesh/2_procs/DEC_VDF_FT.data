# Boiling bubble simulation #
# PARALLEL OK #
dimension 3

domaine dom
domaine dom_solide
Probleme_FT_Disc_gen pb
Pb_conduction pb2

# BEGIN MESH #
Mailler dom
{  
 	pave pave1
	{ 
	origine 0. 0. 0.
	longueurs 1. 1. 1. 
	nombre_de_noeuds 21 21 41
	}
	{
	bord paroi1   			 X = 0.	    	0. <= Y <= 1.		0. <= Z <= 1.
	bord haut1    			 Z = 1.   	0. <= X <= 1.		0. <= Y <= 1.
	raccord local homogene bas1      Z = 0.	    	0. <= X <= 1.		0. <= Y <= 1.
	bord paroi1  			 X = 1.   	0. <= Y <= 1.		0. <= Z <= 1.
	bord paroi1   			 Y = 0.	    	0. <= X <= 1.		0. <= Z <= 1.
	bord paroi1    			 Y = 1.  	0. <= X <= 1.		0. <= Z <= 1.
	}
}
transformer dom x*0.008-0.004 y*0.008-0.004 z*0.016
/* raffiner_isotrope dom */
Mailler dom_solide
{  
 	pave pave2
	{ 
	origine 0. 0. 0.
	longueurs 1. 1. 1. 
	nombre_de_noeuds 21 21 3
	
	}
	{
	bord paroi2    			 X = 0.	    	0. <= Y <= 1.		0. <= Z <= 1.
	raccord local homogene haut2     Z = 1.   	0. <= X <= 1.		0. <= Y <= 1.
	bord bas2     			 Z = 0.	    	0. <= X <= 1.		0. <= Y <= 1.
	bord paroi2   			 X = 1.   	0. <= Y <= 1.		0. <= Z <= 1.
	bord paroi2   			 Y = 0.	    	0. <= X <= 1.		0. <= Z <= 1.
	bord paroi2    			 Y = 1.  	0. <= X <= 1.		0. <= Z <= 1.
	}
}
transformer dom_solide x*0.008-0.004 y*0.008-0.004 z*0.0001-0.0001
/* raffiner_isotrope dom_solide */
# END MESH #
# BEGIN PARTITION #
Partition dom
{
	Partition_tool metis { Nb_parts 2 }
	Nb_parts_tot 2		
	Larg_joint 2
	Nom_Zones DOM
}

Partition dom_solide
{
	Partition_tool metis { Nb_parts 2 }		
	Larg_joint 2
	Nom_Zones DOM_SOLIDE
}
Fin
# END PARTITION #

# BEGIN SCATTER
Scatter DOM.Zones dom
Scatter DOM_SOLIDE.Zones dom_solide
END SCATTER #
    
Sous_Zone zone_destruction
Associer zone_destruction dom
Lire zone_destruction { Boite Origine -0.004 -0.004 0.01568 Cotes 0.008 0.008 0.00016 }

Sous_Zone zone_tranche
Associer zone_tranche dom
Lire zone_tranche { Boite Origine -0.004 -0.004 0.01368 Cotes 0.008 0.008 0.00016 }

Fluide_Incompressible  eau
Lire eau 
{
  	mu champ_uniforme       1 0.00028291
  	rho champ_uniforme      1 958
  	lambda champ_uniforme   1 0.67909
  	cp champ_uniforme       1 4215.7
  
  
}

Fluide_Incompressible vapeur
Lire vapeur
{
  mu champ_uniforme       1 1.2269e-05	
  rho champ_uniforme      1 0.59817
  lambda champ_uniforme   1 0.025096	
  cp champ_uniforme       1 2.0800e3
 
}

Fluide_diphasique fluide
Lire fluide
{
  fluide1 eau
  fluide0 vapeur
  sigma   Champ_Uniforme 1            0.058912	
  chaleur_latente Champ_Uniforme 1    -2256.43e3
}

Solide sol
Read sol
{
	rho Uniform_Field 1 8500.
	lambda Champ_Uniforme 1 234.    
	Cp Champ_Uniforme 1 376.
}


Schema_Euler_explicite  sch
Lire sch
{
	nb_pas_dt_max 10
	tinit 1.272135e-01
	tmax  2.
	dt_min 1.e-7
	dt_max 1.e-5
        dt_impr 1e-17
	dt_sauv 4e-3
	seuil_statio -1
}

Champ_Uniforme gravite
Lire gravite 3 0. 0. -9.81
Associate fluide gravite

Navier_stokes_FT_disc                         hydraulique
Transport_interfaces_FT_disc                  interf
convection_diffusion_temperature_ft_disc      thermique
convection_diffusion_temperature_ft_disc      thermique_vapeur
Associate pb dom
Associate pb hydraulique
Associate pb thermique
Associate pb thermique_vapeur
Associate pb interf
Associate pb fluide

Associate pb2 dom_solide
Associate pb2 sol
VDF dis

Probleme_Couple pbc
Associate pbc pb
Associate pbc pb2
Associate pbc sch
Discretize pbc dis

system "mkdir -p lata"
Lire pb
{
	
	hydraulique
	{
	 	modele_turbulence sous_maille_wale
        	{
         	  Cw               1.e-16
         	  turbulence_paroi negligeable
        	}
		solveur_pression GCP { optimized precond ssor { omega 1.5 } seuil 1e-12 impr }
		convection { quick }
		diffusion { }
		conditions_initiales { vitesse champ_uniforme 3 0. 0. 0. }
		
		equation_interfaces_proprietes_fluide interf
		equation_temperature_mpoint thermique
		equation_temperature_mpoint_vapeur thermique_vapeur
                correction_courbure_ordre 2
		boundary_conditions 
		{
		paroi1   paroi_fixe 
		bas1     paroi_fixe 
		haut1    sortie_libre_rho_variable champ_front_uniforme 1 0.
        	}
	}
	interf
	{
		interpolation_repere_local
		methode_transport vitesse_interpolee hydraulique
	        conditions_initiales { fonction

(x^2+y^2+(z-(0.000848)*(0.4573484))^2-0.000848^2)

} 
		iterations_correction_volume 2
 methode_interpolation_v vdf_lineaire
        	n_iterations_distance 10
		remaillage {
			pas 1e-6 
			nb_iter_remaillage 1
			critere_arete 0.35
			critere_remaillage 0.2
			pas_lissage 0.000001
			lissage_courbure_iterations_systematique 10 # 1 #
			lissage_courbure_iterations_si_remaillage 10 # 1 #
			lissage_courbure_coeff -0.05
			nb_iter_barycentrage 5 # 5 #
			relax_barycentrage 1
			facteur_longueur_ideale 2.
			nb_iter_correction_volume 3
			seuil_dvolume_residuel 1e-12
			}
			#
		 remaillage {
  pas 1.e-4
  nb_iter_remaillage 2
  critere_arete 0.35
  pas_lissage 1.e-3
  lissage_courbure_iterations_systematique 1
  lissage_courbure_iterations_si_remaillage 3
  lissage_courbure_coeff -0.05
  nb_iter_barycentrage 2
  relax_barycentrage 1.00
  facteur_longueur_ideale 1.45
  nb_iter_correction_volume 8
  seuil_dvolume_residuel 1e-20
			}
			#
		collisions
		{
                active
		Juric_local
                type_remaillage thomas { distance_interface_element_max 2 }
  phase_continue 1
 }
		boundary_conditions
		{
		bas1 Paroi_FT_disc constant champ_front_fonc_xyz 1 55.*((sqrt(x*x+y*y))_GT_1.0e-3)+120.*((sqrt(x*x+y*y))_LT_0.85e-3)
		paroi1 Paroi_FT_disc symetrie
		haut1   Paroi_FT_disc symetrie
		} #
                maillage                                                                                                           
                {                                                                                                                  
                        niveau_plot 1                                                                                              
                        correction_contact_courbure_coeff 2.0                                                                      
                        calcul_courbure_iterations 2                                                                               
                        niter_pre_lissage 1                                                                                       
                        methode_calcul_courbure_contact_line hysteresis                                                            
                        weight_CL                         0.0                                                                      
                } #
                parcours_interface { Correction_Parcours_Thomas }
 		suppression_sous_zone zone_destruction

	}

        thermique
        {
                equation_interface interf
                equation_navier_stokes hydraulique
                correction_courbure_ordre 2
                phase 1
		# maintien_temperature zone_tranche -6. #
                stencil_width 10
                diffusion { }
                convection { quick }
                boundary_conditions
                {
			bas1   echange_contact_VDF_FT_Disc {
					autre_probleme pb2
					autre_bord haut2 
					autre_champ_temperature temperature 
					nom_mon_indicatrice indicatrice_interf 
					phase 1
				}  
                        haut1  frontiere_ouverte T_ext Champ_front_Uniforme 1 -6.
                        paroi1 frontiere_ouverte T_ext Champ_front_Uniforme 1 -6. # paroi_adiabatique #
                }
                conditions_initiales { Temperature_thermique champ_fonc_xyz dom 1 -6.+(z<20e-4)*(12.*(1-z/20e-4)) }
        }
	
	thermique_vapeur
	{
		equation_interface interf
		equation_navier_stokes hydraulique
                correction_courbure_ordre 2
		phase 0
		stencil_width 10
                diffusion { }
                convection { quick }
                boundary_conditions
                {
			bas1   echange_contact_VDF_FT_Disc {
					autre_probleme pb2
					autre_bord haut2 
					autre_champ_temperature temperature 
					nom_mon_indicatrice indicatrice_interf 
					phase 0
				}
			haut1  frontiere_ouverte T_ext Champ_front_Uniforme 1 0.
			paroi1 paroi_adiabatique
                }
                conditions_initiales { Temperature_thermique_vapeur champ_fonc_xyz dom 1 0. }
	}
	postraitement 
	{
		champs dt_post 1.e-3
		{
			vitesse elem
			temperature_thermique elem
		}
	}
	
liste_postraitements
{
	
   Postraitement_ft_lata post1
   {
	dt_post 1.e-3
	nom_fichier lata/post
 	champs elements
  	      {
  		 
  		 indicatrice_interf
  		 temperature_thermique
		 temperature_thermique_vapeur
		 vitesse
		 pression
		 temperature_mpoint
		 temperature_mpointv
  	      }
	      
		
	interfaces interf
	      {  
		champs sommets { courbure }
		champs elements { PE PE_LOCAL COMPO_CONNEXE }

              }

    }
    }
# Resume_last_time binaire ../PAR_FTD_Boiling_bubble_pb.sauv  #
}


Read pb2
{
	Conduction
	{
		diffusion { }
		initial_conditions {
			temperature Champ_Uniforme 1 6.
			}
		boundary_conditions {
			paroi2 symetrie
			bas2 paroi_flux_impose Champ_front_fonc_xyz 1 2471144*exp(-(x*x+y*y)/(3.6067*10^(-7)))
			haut2  echange_contact_VDF_FT_Disc_solid {
				autre_probleme pb
				autre_bord bas1 
				autre_champ_temperature_indic1 temperature_thermique
				autre_champ_temperature_indic0 temperature_thermique_vapeur
				autre_champ_indicatrice indicatrice_interf 
			}

		}
	}

	Post_processing
	{
		Format lml
		Definition_champs 
		{
			diffusion_Ts operateur_eqn 
			{ 
				sources { refchamp { pb_champ pb2 temperature  } }  
				numero_op 0
			}
		}
		fields dt_post 1.e-3
		{
		    diffusion_Ts elem
		    temperature elem
		}
	}	
}
Solve pbc
FIN

			     
