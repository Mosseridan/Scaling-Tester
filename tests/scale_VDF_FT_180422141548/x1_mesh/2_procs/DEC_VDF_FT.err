Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VDF_FT
feenableexcept enabled.
Data file : DEC_VDF_FT
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VDF_FT_180422-141548/x1_mesh/2_procs
Nom du cas DEC_VDF_FT
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block pave1
Step of 3D mesh in progress... 
It was created 5% of coordxIt was created 10% of coordxIt was created 15% of coordxIt was created 20% of coordxIt was created 25% of coordxIt was created 30% of coordxIt was created 35% of coordxIt was created 40% of coordxIt was created 45% of coordxIt was created 50% of coordxIt was created 55% of coordxIt was created 60% of coordxIt was created 65% of coordxIt was created 70% of coordxIt was created 75% of coordxIt was created 80% of coordxIt was created 85% of coordxIt was created 90% of coordxIt was created 95% of coordx
It was created 5% of coordyIt was created 10% of coordyIt was created 15% of coordyIt was created 20% of coordyIt was created 25% of coordyIt was created 30% of coordyIt was created 35% of coordyIt was created 40% of coordyIt was created 45% of coordyIt was created 50% of coordyIt was created 55% of coordyIt was created 60% of coordyIt was created 65% of coordyIt was created 70% of coordyIt was created 75% of coordyIt was created 80% of coordyIt was created 85% of coordyIt was created 90% of coordyIt was created 95% of coordy
It was created 2% of coordzIt was created 5% of coordzIt was created 7% of coordzIt was created 10% of coordzIt was created 12% of coordzIt was created 15% of coordzIt was created 17% of coordzIt was created 20% of coordzIt was created 22% of coordzIt was created 25% of coordzIt was created 27% of coordzIt was created 30% of coordzIt was created 32% of coordzIt was created 35% of coordzIt was created 37% of coordzIt was created 40% of coordzIt was created 42% of coordzIt was created 45% of coordzIt was created 47% of coordzIt was created 50% of coordzIt was created 52% of coordzIt was created 55% of coordzIt was created 57% of coordzIt was created 60% of coordzIt was created 62% of coordzIt was created 65% of coordzIt was created 67% of coordzIt was created 70% of coordzIt was created 72% of coordzIt was created 75% of coordzIt was created 77% of coordzIt was created 80% of coordzIt was created 82% of coordzIt was created 85% of coordzIt was created 87% of coordzIt was created 90% of coordzIt was created 92% of coordzIt was created 95% of coordzIt was created 97% of coordz
Step of mesh ended 
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Reading of the boundary haut1
End of reading of the boundary : haut1
Reading of the boundary bas1
End of reading of the boundary : bas1
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Reading of the boundary paroi1
End of reading of the boundary : paroi1
Compressing objects of type Zone in progress... 
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x*0.008-0.004
Interpretation of the function x*0.008-0.004 Ok
Reading and interpretation of the function y*0.008-0.004
Interpretation of the function y*0.008-0.004 Ok
Reading and interpretation of the function z*0.016
Interpretation of the function z*0.016 Ok
Hexaedre::reordonner : 0 elements reversed
Hexaedre::reordonner : 0 elements reversed
Execution of the interpreter Mailler
Association of the domain on which the interpreter applies Mailler
Reading of the block pave2
Step of 3D mesh in progress... 
It was created 5% of coordxIt was created 10% of coordxIt was created 15% of coordxIt was created 20% of coordxIt was created 25% of coordxIt was created 30% of coordxIt was created 35% of coordxIt was created 40% of coordxIt was created 45% of coordxIt was created 50% of coordxIt was created 55% of coordxIt was created 60% of coordxIt was created 65% of coordxIt was created 70% of coordxIt was created 75% of coordxIt was created 80% of coordxIt was created 85% of coordxIt was created 90% of coordxIt was created 95% of coordx
It was created 5% of coordyIt was created 10% of coordyIt was created 15% of coordyIt was created 20% of coordyIt was created 25% of coordyIt was created 30% of coordyIt was created 35% of coordyIt was created 40% of coordyIt was created 45% of coordyIt was created 50% of coordyIt was created 55% of coordyIt was created 60% of coordyIt was created 65% of coordyIt was created 70% of coordyIt was created 75% of coordyIt was created 80% of coordyIt was created 85% of coordyIt was created 90% of coordyIt was created 95% of coordy
It was created 50% of coordz
Step of mesh ended 
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Reading of the boundary haut2
End of reading of the boundary : haut2
Reading of the boundary bas2
End of reading of the boundary : bas2
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Reading of the boundary paroi2
End of reading of the boundary : paroi2
Compressing objects of type Zone in progress... 
Zones::comprimer() - End
Elimination of 0 unnecessary nodes in the domain dom_solide
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x*0.008-0.004
Interpretation of the function x*0.008-0.004 Ok
Reading and interpretation of the function y*0.008-0.004
Interpretation of the function y*0.008-0.004 Ok
Reading and interpretation of the function z*0.0001-0.0001
Interpretation of the function z*0.0001-0.0001 Ok
Hexaedre::reordonner : 0 elements reversed
Hexaedre::reordonner : 0 elements reversed
Decouper : Splitting of a domain
 Domain name to split : dom
==============================================
The extreme coordinates of the domain dom are:
x is between -0.004 and 0.004
y is between -0.004 and 0.004
z is between 0 and 0.016
==============================================
==============================================
The volume cells of the domain dom are:
sum(volume cells)= 1.024e-06
mean(volume cells)= 6.4e-11
min(volume cells)= 6.4e-11
max(volume cells)= 6.4e-11
==============================================
domaine = dom
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 400
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 1
The partitioner has generated 2 parts.
Number of parts requested : 2
Generation of 0 empty parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 2 parts:
 Construction of part number 0
  Number of nodes    : 9261
  Number of elements : 8000
  Number of joints   : 1
  Joint 0 PeVoisin 1 NbSommets 441 NbFaces 400 NbElemDist 800
               Total:    NbSommets 441 NbFaces 400 NbElemDist 800
Writing part 0 into the binary file DOM_0000.Zones
Writing of 9261 nodes.
Writing of 8000 elements.
 Construction of part number 1
  Number of nodes    : 9261
  Number of elements : 8000
  Number of joints   : 1
  Joint 0 PeVoisin 0 NbSommets 441 NbFaces 400 NbElemDist 800
               Total:    NbSommets 441 NbFaces 400 NbElemDist 800
Writing part 1 into the binary file DOM_0001.Zones
Writing of 9261 nodes.
Writing of 8000 elements.
End of the interpreter Decouper
Decouper : Splitting of a domain
 Domain name to split : dom_solide
==============================================
The extreme coordinates of the domain dom_solide are:
x is between -0.004 and 0.004
y is between -0.004 and 0.004
z is between -0.0001 and 0
==============================================
==============================================
The volume cells of the domain dom_solide are:
sum(volume cells)= 6.4e-09
mean(volume cells)= 8e-12
min(volume cells)= 8e-12
max(volume cells)= 8e-12
==============================================
domaine = dom_solide
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 46
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 1
The partitioner has generated 2 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 2 parts:
 Construction of part number 0
  Number of nodes    : 699
  Number of elements : 400
  Number of joints   : 1
  Joint 0 PeVoisin 1 NbSommets 72 NbFaces 46 NbElemDist 92
               Total:    NbSommets 72 NbFaces 46 NbElemDist 92
Writing part 0 into the binary file DOM_SOLIDE_0000.Zones
Writing of 699 nodes.
Writing of 400 elements.
 Construction of part number 1
  Number of nodes    : 696
  Number of elements : 400
  Number of joints   : 1
  Joint 0 PeVoisin 0 NbSommets 72 NbFaces 46 NbElemDist 92
               Total:    NbSommets 72 NbFaces 46 NbElemDist 92
Writing part 1 into the binary file DOM_SOLIDE_0001.Zones
Writing of 696 nodes.
Writing of 400 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
