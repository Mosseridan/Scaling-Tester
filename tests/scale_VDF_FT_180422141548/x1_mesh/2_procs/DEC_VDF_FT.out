-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VDF_FT
 
Reading data file 

14 MBytes of RAM taken by the calculation.
15 MBytes of RAM taken by the calculation.
16 MBytes of RAM taken by the calculation.
16 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 2
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[5.00e-01]   1=[5.00e-01]
   Allowed maximum load imbalance: 1.001 

     16000      92000      92000 [20] [     1200:   16000 ]
      8371      67350      76742 [20] [     1200:   16000 ]
      4383      43828      64964 [20] [     1200:   16000 ]
      2296      26120      55448 [20] [     1200:   16000 ]
      1194      14656      46956 [20] [     1200:   16000 ]
       622       7840      39796 [20] [     1200:   16000 ]
       322       4070      33642 [20] [     1200:   16000 ]
       165       2004      28164 [20] [     1200:   16000 ]
        84        964      23324 [20] [     1200:   16000 ]
        44        460      19158 [20] [     1200:   16000 ]
        23        194      15168 [20] [     1200:   16000 ]
        20        158      14698 [20] [     1200:   16000 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.012
     Coarsening: 		   0.009
            Matching: 			   0.000
            Contract: 			   0.008
     Initial Partition: 	   0.000
     Uncoarsening: 		   0.003
          Refinement: 			   0.002
          Projection: 			   0.002
     Splitting: 		   0.000
********************************************************************

 gk_mcore statistics
           coresize:       192192         nmops:         2048  cmop:      0
        num_callocs:          118   num_hallocs:            0
       size_callocs:       847768  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       192016   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 2
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[5.00e-01]   1=[5.00e-01]
   Allowed maximum load imbalance: 1.001 

       800       3840       3840 [20] [       60:     800 ]
       420       2660       3080 [20] [       60:     800 ]
       220       1594       2488 [20] [       60:     800 ]
       116        830       2020 [20] [       60:     800 ]
        63        396       1612 [20] [       60:     800 ]
        33        174       1166 [20] [       60:     800 ]
        19         84        888 [20] [       60:     800 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.000
     Coarsening: 		   0.000
            Matching: 			   0.000
            Contract: 			   0.000
     Initial Partition: 	   0.000
     Uncoarsening: 		   0.000
          Refinement: 			   0.000
          Projection: 			   0.000
     Splitting: 		   0.000
********************************************************************

 gk_mcore statistics
           coresize:         9792         nmops:         2048  cmop:      0
        num_callocs:           73   num_hallocs:            0
       size_callocs:        45224  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:         9616   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

18 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 0.246051 s
